src/LiveComponent/src/Util/LiveControllerAttributesCreator.php

```diff
@@ -108,6 +108,18 @@ public function attributesForRendering(MountedComponent $mounted, ComponentMetad
             }
         }
 
+        $liveMetadata = $this->metadataFactory->getMetadata($mounted->getName());
```

Comment:
Can a LiveComponent be a child component AND use urlquery string ?

Reply:
Yes it can, what are the implications you'd have in mind here?

Reply:
Just wanted to see if we could early exit somewhere.

But otherwise, we can move thus line before L101 and reuse $liveMetadata in the calculateFingerprint call ?